security/pkg/stsservice/tokenmanager/google/tokenexchangeplugin.go

```diff
@@ -135,7 +167,18 @@ func (p *Plugin) constructFederatedTokenRequest(parameters stsservice.StsRequest
 	req, _ := http.NewRequest("POST", federatedTokenEndpoint, bytes.NewBuffer(jsonQuery))
 	req.Header.Set("Content-Type", contentType)
 	if pluginLog.DebugEnabled() {
-		reqDump, _ := httputil.DumpRequest(req, false)
+		dQuery := map[string]string{
+			"audience":           aud,
+			"grantType":          parameters.GrantType,
+			"requestedTokenType": tokenType,
+			"subjectTokenType":   parameters.SubjectTokenType,
+			"subjectToken":       "redacted",
```

Comment:
"redacted". Is this intended?

Reply:
This is intended. This is only used in debug log.